azext_iot/constants.py

```diff
@@ -48,3 +48,6 @@
 
 # Config Key's
 CONFIG_KEY_UAMQP_EXT_VERSION = "uamqp_ext_version"
+
+# Initial Track 2 SDK version
+TRACK_2_SDK_MIN_VERSION = '1.0.0'
```

Comment:
We should prefix this with `IOTHUB_` or re-organize like creating dictionary that holds IoT Hub constants